src/logging.cc

```diff
@@ -1620,9 +1620,10 @@ void LogMessage::Init(const char* file,
     #ifdef GLOG_CUSTOM_PREFIX_SUPPORT
       if (custom_prefix_callback == NULL) {
     #endif
-          stream() << LogSeverityNames[severity][0]
-                   << setw(4) << 1900 + logmsgtime_.year()
-                   << setw(2) << 1 + logmsgtime_.month()
+          stream() << LogSeverityNames[severity][0];
+          if (FLAGS_log_year_in_prefix)
```

Comment:
Please add parens here (see Google style guide).

Reply:
What is "parens" ? I searched it here: https://google.github.io/styleguide/cppguide.html and I did not find it. Would you please advise.